astronomer/airflow/version_check/update_checks.py

```diff
@@ -81,7 +90,7 @@ def run(self):
             try:
                 update_available, wake_up_in = self.check_for_update()
                 if update_available == UpdateResult.SUCCESS_UPDATE_AVAIL:
-                    self.log.info("A new version of Astronomer Certified Airflow is available")
+                    self.log.info("A new version of %s is available", _app_name())
```

Comment:
The "Airflow" part will fall off with this change ("Astronomer Certified Airflow" -> "Astronomer Certified"). Is this intentional?

Reply:
Was intentional but I think I can do better. Fixing...